cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1050,55 +1050,6 @@ def default_for(
 
                     .. versionadded:: 8.0.0
                 """)
-
-    with Conf('editors', desc='''
```

Comment:
Do you want to provide obseletion notices for these in the upgrader?

Reply:
Good point, I suppose I should!

Reply:
Done.